srttools/monitor/monitor.py

```diff
@@ -8,23 +7,23 @@
 import threading
 import time
 import warnings
-
+import re
+import multiprocessing as mp
 from srttools.imager import main_preprocess
-from srttools.monitor.common import MAX_FEEDS, exit_function, log
+from srttools.monitor.common import exit_function, log
 from srttools.read_config import read_config
 from srttools.scan import product_path_from_file_name
 
 try:
-    from watchdog.events import FileMovedEvent, PatternMatchingEventHandler
+    from watchdog.events import FileMovedEvent, RegexMatchingEventHandler
```

Comment:
I switched from Pattern to Regex in order to match an arbitrary number of X in fitsX files.
With the PatternMatchingEventHandler the pattern would be just some repetitions of [0-9][0-9]...[0-9] and so on.